api/handlers_test.go

```diff
@@ -136,6 +139,44 @@ func TestCreateSessionHandlerFunc(t *testing.T) {
 	})
 }
 
+func TestGetSessionHandlerFunc(t *testing.T) {
+	Convey("Given a request to retrieve a stored session", t, func() {
```

Comment:
Think of these statements like BDD statements - they describe the behaviour but don't need to be too technical for example _Given a valid request_